add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The HTML and PHP code are mixed, which can lead to maintenance challenges.

Recommendations:
- **Separate Concerns**: Use a templating engine like Twig or Blade to separate HTML from PHP logic. This improves readability and maintainability.

Example:
```php
// In add_book.php
include 'template_engine.php';

$template = $templateEngine->load('add_book.html');
$template->render(['title' => $title, 'author' => $author, 'year' => $year]);
```

// In add_book.html
```html
<h1>Add a New Book</h1>
<form method="POST">
    <label>Title:</label>
    <input type="text" name="title" value="{{ title }}"><br>
    <label>Author:</label>
    <input type="text" name="author" value="{{ author }}"><br>
    <label>Year:</label>
    <input type="number" name="year" value="{{ year }}"><br>
    <button type="submit">Add Book</button>
</form>
```